lms/templates/course_experience/course-outline-fragment.html

```diff
@@ -0,0 +1,249 @@
+## mako
+
+<%page expression_filter="h"/>
+
+<%namespace name='static' file='../static_content.html'/>
+
+<%!
+from datetime import date
+
+from django.utils.translation import ugettext as _
+
+from openedx.core.djangolib.markup import HTML, Text
+
+from opaque_keys.edx.keys import CourseKey
+%>
+
+<%
+course_sections = blocks.get('children')
+%>
+
+<%
+    staff = None
+    if 'masquerade_settings' in request.session:
+        course_key = CourseKey.from_string(request.session['course_id']) if request.session['masquerade_settings'] else None
+
+        if not request.session['masquerade_settings'] and not course_key:
+            staff = 'staff'
```

Comment:
Could you change the name of this var? It is a little bit confusing when staff is not staff.

Reply:
done